Simplenote/src/main/res/layout/activity_notes.xml

```diff
@@ -40,7 +40,8 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:orientation="vertical"
-            android:paddingTop="@dimen/nav_drawer_navigation_padding_top">
```

Comment:
Now that `nav_drawer_navigation_padding_top` is no longer being used, we can remove it from `values/dimens.xml` and `values-v21/dimens.xml`.  Since that was the only resource in `values-v21/dimens.xml`, we can remove that file entirely.